### DISCUSSION TITLE
update container with new image tag

### DISCUSSION THREAD

**Discussion by weironz** [containrrr/watchtower#621] [Q&A] [answered]
## question
i run docker build in jenkins cicd, then push the image to harbor , every image have a new image tag with BUILD_ID env, then i will update the image in remote docker hosts by docker pull and docker run。

## what i want
when i push a new image with new tag to harbor registry , watchtower update a container with the new tag，but watchtower only suppport update a fixed tag？

and if you always use the same tag ,  It's impossible when you want to perform a rollback some day。

**Comment by simonaronsson** [comment, marked as answer]
Hi and welcome to the community!

Yes, watchtower only allows for tracking a fixed tag. At the moment, we have no plans to introduce features that allow for following changing tag names. The usual approach is to use `latest` as a static tag and point it at the same images as your versioned tags.

**Comment by weironz** [comment]
ok, but if it  can following changing tag names in harbor or some else registry , it will be a good feature，then watchtower can intergration into cicd like jenkins ,  and Rolling update a running container by a Elegant way。

usually cicd will  auto build a image with tag like BUILD_ID 1 2 3 4.... v1.0 v2.0 v3.0  or git COMMIT_ID ， maybe watchtower can following a image tag by the Regular number or sort them by the latest time.

**Reply by piksel** [reply]
No, that doesn't work unfortunately, but you should be able to specify additional tags in your CI.
